client/e2e/pages/EventsPage.ts

```diff
@@ -82,7 +89,7 @@ export class EventsPage {
     this.activeViewOption = page.getByTestId('active-button');
     this.inactiveViewOption = page.getByTestId('inactive-button');
     this.allViewOption = page.getByTestId('all-button');
-
+    this.deleteButton = page.getByTestId('event-delete-button');

```

Comment:
why use this when a deleteEventButton exists and is the same? make sure you use the deleteEventButton as it a more specifically named locator. (get rid of this one)